### DISCUSSION TITLE
Is there a way to have a recurring task show in the calendar for each recurrrence?

### DISCUSSION THREAD

**Discussion by mrchockablock** [ivan-lednev/obsidian-day-planner#529] [Q&A]
Hi there,

Firstly I'm just discovering Obsidian and hence Day Planner too - but it looks great! I wondered if there's a way to create a recurring task?

eg. If I want to "do the funky chicken" every day between 09:00 and 09:30, is there a way to incorporate this into Day Planner?

Thanks in advance for your help.

**Comment by ivan-lednev** [comment]
Hello, right now there is no way to do that.

**Reply by mhdna** [reply]
Hi, 

Just to make sure, so there's still no way to show something like this `- [ ] 21:00 - 22:00 Reading  🔁 every day  #habit` using the tasks plugin? 

I have `#habit` as a dataview source of course.

**Comment by mrchockablock** [comment]
Hey Mahdi,

Try using the @ symbol instead for tagging tasks (versus # for notes) eg.

@next
@shopping

To give each task a context.



Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by mhdna** [reply]
But how would you query it as a dataview source so that they appear in the daily-planner schedule, though? Apparently the engine does not know how to parse these.

![image](https://github.com/user-attachments/assets/e60103ec-4c56-4e31-ac76-27dbf38600bb)
